### DISCUSSION TITLE
The max amount of translated pages per PR

### DISCUSSION THREAD

**Discussion by spageektti** [tldr-pages/tldr#12768] [Q&A] [answered]
For example, let's say I translated 50 pages into Polish from the `linux` directory. 
Should I divide this to multiple PRs and if so how many translated pages per PR?

**Comment by kbdharun** [comment, marked as answer]
We generally suggest a maximum of 10 pages in PRs for additions/modifications, as it allows us to review it throughly.

This isn't a hard cap so you could modify a few more pages too if required.

So I would indeed suggest dividing your 50 translated pages additions to 4 or 5 PRs.
